data/json/mapgen/lake_buildings/cabin_lake.json

```diff
@@ -168,7 +168,7 @@
         "//..////......9........."
       ],
       "place_vehicles": [
-        { "vehicle": "boat_motor_single", "x": 12, "y": 5, "rotation": 180, "chance": 10, "status": 3 },
+        { "vehicle": "lake", "x": 12, "y": 5, "rotation": 180, "chance": 10, "status": 3 },
```

Comment:
this is far too large to fit in that dock and won't spawn even when I try to force it.  If you want it at the lake cabin, anchor it off shore somewhere.

Reply:
Fixed by adding a blank 'lake' tile to the overmap and spawning the lake vehicle group there, the boat_motor_single was re-added